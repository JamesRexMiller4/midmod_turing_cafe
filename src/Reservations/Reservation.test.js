import React from 'react';
import {shallow} from 'enzyme';
import Reservation from './Reservations.js'
 
describe('', () => {
let wrapper, data;
 
beforeEach(() => {

  data = [
    {id:1, name:"Jerry", time:"12:12", date:"12/12", number:4},
    {id:2, name:"Barb", time:"14:14", date:"12/14", number:2},
    {id:3, name:"Murph", time:"1:12", date:"12/12", number:3}
  ]



wrapper = shallow(<Reservation 
  data={data}
/>)
})
 
it('should match the snapshot with all the data passed through', () => {
  expect(wrapper).toMatchSnapshot()
});
 
});