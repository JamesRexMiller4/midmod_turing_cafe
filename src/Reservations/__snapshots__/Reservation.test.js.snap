// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot with all the data passed through 1`] = `
<div
  className="reservations-wrapper-div"
>
  <Card
    date="12/12"
    key="1"
    name="Jerry"
    number={4}
    time="12:12"
  />
  <Card
    date="12/14"
    key="2"
    name="Barb"
    number={2}
    time="14:14"
  />
  <Card
    date="12/12"
    key="3"
    name="Murph"
    number={3}
    time="1:12"
  />
</div>
`;
